### ISSUE TITLE
TCP fast open for webseeds

### ISSUE THREAD

**Issue by f100024** [arvidn/libtorrent#7784]
I did not find where to ask this question, so please forgive me.
I noticed in my case libtorrent is actively using webseed for downloading over http (not https), so does it make sense to use the `TCP fast open` feature?

**Comment by arvidn** [comment]
Yes, that would make sense. It's not so simple though, as it breaks the normal socket abstraction.
